ios/FluentUI/Vnext/List/ListHeaderFooter.swift

```diff
@@ -6,28 +6,44 @@
 import UIKit
 import SwiftUI
 
-/// View for List Header (availble for iOS 14.0+)
 struct Header: View {
-    let title: String
-    var tokens: MSFListTokens
+    @Environment(\.theme) var theme: FluentUIStyle
+    @ObservedObject var state: MSFListSectionState
+    @ObservedObject var tokens: MSFHeaderFooterTokens
 
-    init(title: String, tokens: MSFListTokens) {
-        self.title = title
-        self.tokens = tokens
+    init(state: MSFListSectionState, windowProvider: FluentUIWindowProvider?) {
+        self.state = state
+        self.tokens = MSFHeaderFooterTokens(style: state.style)
+        self.tokens.windowProvider = windowProvider
     }
 
     var body: some View {
         HStack(spacing: 0) {
-            Text(title)
-                .font(Font(tokens.subtitleFont))
-                .foregroundColor(Color(tokens.subtitleColor))
-                .listRowInsets(EdgeInsets())
-                .padding(.top, tokens.horizontalCellPadding / 2)
-                .padding(.leading, tokens.horizontalCellPadding)
-                .padding(.trailing, tokens.horizontalCellPadding)
-                .padding(.bottom, tokens.horizontalCellPadding / 2)
+            if let title = state.title, !title.isEmpty {
+                Text(title)
+                    .font(Font(tokens.textFont))
+                    .foregroundColor(Color(tokens.textColor))
+            }
             Spacer()
         }
         .background(Color(tokens.backgroundColor))
+        .padding(EdgeInsets(top: tokens.topPadding,
+                            leading: tokens.leadingPadding,
+                            bottom: tokens.bottomPadding,
+                            trailing: tokens.trailingPadding))
+        .listRowInsets(EdgeInsets())
```

Comment:
As discussed offline, if this is needed only if we use the Header SwiftUI type in iOS13, we should remove it from code now and document it (out of the code).

Reply:
Just noticed your last push. Thanks for fixing this!